api/v2/urls.py

```diff
@@ -3,7 +3,7 @@
 from api.v2 import views
 
 router = routers.DefaultRouter(trailing_slash=False)
-router.register(r'allocations', views.AllocationViewSet)
+router.register(r'allocations', views.AllocationViewSet, base_name='allocation')
```

Comment:
I don't think this is necessary since the model name matches "allocation" already.  "images" needed the base_name='application' because the model was called application.